apps/rush-lib/src/scripts/install-run-rush.ts

```diff
@@ -23,8 +23,13 @@ import {
 } from './install-run';
 
 const PACKAGE_NAME: string = '@microsoft/rush';
+const RUSH_PREVIEW_VERSION: string = 'RUSH_PREVIEW_VERSION';
 
 function getRushVersion(): string {
+  if (process.env[RUSH_PREVIEW_VERSION] !== undefined) {
```

Comment:
Can you stick the value from the environment variable in a temporary variable so we don't have the same indexer expression twice?